### PR TITLE
Fix linux unit tests

### PR DESCRIPTION
**Pull request by lucacarniato**
Sort contacts by mesh2d indices to get consistent results

### CODE REVIEW

File: tests/test_mesh1d_basics.py [hunk 1]
```diff
@@ -114,11 +131,30 @@ def test_contacts_compute_multiple():
...
+
+    assert contacts.mesh2d_indices[0] == 0
+    assert contacts.mesh2d_indices[1] == 1
+    assert contacts.mesh2d_indices[2] == 6
+    assert contacts.mesh2d_indices[3] == 7
+    assert contacts.mesh2d_indices[4] == 12
+    assert contacts.mesh2d_indices[5] == 13
+    assert contacts.mesh2d_indices[6] == 18
+    assert contacts.mesh2d_indices[7] == 19
+    assert contacts.mesh2d_indices[8] == 24
```
Comment (Hofer-Julian) [inline tests/test_mesh1d_basics.py:157]: Why did you remove the assert_array_equal?

**Comment by Hofer-Julian** [comment]
I would suggest to remove the sorting function and replace all tests where the actual order is not important with something like "assert [1,2] in array.tolist()".

See https://stackoverflow.com/a/14766816/11038610

**Comment by lucacarniato** [comment]
> I would suggest to remove the sorting function and replace all tests where the actual order is not important with something like
> 
> ```python
> assert [1,2] in array.tolist()"
> ```
> 
> See https://stackoverflow.com/a/14766816/11038610
> 
> Or alternatively:
> 
> ```python
> assert 1 in array
> assert 2 in array
> ...
> ```

We still are interested that the 2d faces are connected correctly to the 1d nodes

**Comment by Hofer-Julian** [comment]
> 
> 
> > I would suggest to remove the sorting function and replace all tests where the actual order is not important with something like
> > ```python
> > assert [1,2] in array.tolist()"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > See https://stackoverflow.com/a/14766816/11038610
> > Or alternatively:
> > ```python
> > assert 1 in array
> > assert 2 in array
> > ...
> > ```
> 
> We still are interested that the 2d faces are connected correctly to the 1d nodes

Ah yes good point!
